backend/core/detection.py

```diff
@@ -769,6 +769,18 @@ def commit_hijack(
                     result = hijack_value
                     mail_log.info("{}".format(result))
                 redis_pipeline.set(redis_hijack_key, pickle.dumps(result))
+
+                # store the origin, neighbor combination for this hijack BGP update
+                origin = None
+                neighbor = None
+                if monitor_event["path"]:
+                    origin = monitor_event["path"][-1]
+                if len(monitor_event["path"]) > 1:
+                    neighbor = monitor_event["path"][-2]
```

Comment:
what happens in this case? what is stored eventually and how we can use it?

Reply:
Addressed in offline discussion (possible pairs: (origin, None), (origin, neighbor)). None values will be addressed in post-processing.